examples/java-matter-controller/java/src/com/matter/controller/commands/pairing/PairOnNetworkLongImReadCommand.kt

```diff
@@ -35,9 +37,89 @@ class PairOnNetworkLongImReadCommand(
       setFailure("read failure")
     }
 
+    fun checkAttributeTlv(attribute: AttributeState): Boolean {
```

Comment:
I wonder if more kotlin-like to have:

```kt
fun checkAttributeTlv(attribute: AttributeState): Boolean = 
   attribute.getTlv().joinToString().equals("""21, 40, 16, 24""").also {
     if (!it) {
        logger.log(Level.INFO, "read attribute tlv failure")
     }
   };
```

Maybe even with a helper so one can call:

```kt
//...
   = attribute.getTlv().joinToString().equals("""21, 40, 16, 24""").logIfFalse("read attribute tlv failure")
```

Reply:
done, thanks